cmd/preflight/cli/stdout_results.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+
+	analyzerunner "github.com/replicatedhq/troubleshoot/pkg/analyze"
+)
+
+var failed = false
```

Comment:
Why the global variable and not something like having `outputResult` return the failure status?